src/main/java/aqua/aquatask/AquaDeadline.java

```diff
@@ -0,0 +1,77 @@
+package aqua.aquatask;
+
+import java.time.LocalDateTime;
+import java.util.Optional;
```

Comment:
I have noticed your usage of Optionals. I believe this is a good practice to prevent null exceptions. Also, this shows your competency in java programming